tsconfig.json

```diff
@@ -28,6 +28,7 @@
     // Must be in sync with jest.config.js >> moduleNameMapper
     "paths": {
       "Components/*": ["./src/script/components/*"],
+      "Modals/*": ["./src/script/page/Modals/*"],
```

Comment:
I'd like to start a discussion about this. 
I don't think we should add aliases too lightly. Aliases are great for utility tools that we use across the entire app, not for isolated things we use in 2/3 different places. 
Aliases tie us a lot with the bundler/compilation step we use and we need to maintain those across all build steps (tsconfig, webpack config, test config...). 

In this case, I would rather not add a custom alias for this